packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -249,6 +249,283 @@ Array [
+            </a>
+          </td>
+          <td
+            className="pf-c-table__action"
+            data-key={3}
+            data-label={null}
+            onMouseEnter={[Function]}
+            style={
+              Object {
+                "paddingRight": 0,
+              }
+            }
+          >
+            <div
+              className="pf-c-dropdown pf-m-align-right"
+              data-ouia-component-id="OUIA-Generated-Dropdown-7"
+              data-ouia-component-type="PF4/Dropdown"
+              data-ouia-safe={true}
+            >
+              <button
+                aria-expanded={false}
+                aria-haspopup={true}
+                aria-label="Actions"
+                className="pf-c-dropdown__toggle pf-m-plain"
+                disabled={true}
+                id="pf-dropdown-toggle-id-6"
+                onClick={[Function]}
+                onKeyDown={[Function]}
+                type="button"
+              >
+                <svg
+                  aria-hidden={true}
+                  aria-labelledby={null}
+                  fill="currentColor"
+                  height="1em"
+                  role="img"
+                  style={
+                    Object {
+                      "verticalAlign": "-0.125em",
+                    }
+                  }
+                  viewBox="0 0 192 512"
+                  width="1em"
+                >
+                  <path
+                    d="M96 184c39.8 0 72 32.2 72 72s-32.2 72-72 72-72-32.2-72-72 32.2-72 72-72zM24 80c0 39.8 32.2 72 72 72s72-32.2 72-72S135.8 8 96 8 24 40.2 24 80zm0 352c0 39.8 32.2 72 72 72s72-32.2 72-72-32.2-72-72-72-72 32.2-72 72z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </td>
+        </tr>
+        <tr
+          className=""
+          data-ouia-component-id="OUIA-Generated-TableRow-10"
+          data-ouia-component-type="PF4/TableRow"
+          data-ouia-safe={true}
+          hidden={false}
+          onClick={[Function]}
+          onKeyDown={[Function]}
+        >
+          <td
+            className="pf-c-table__check"
+            data-key={0}
+            data-label={null}
+            onMouseEnter={[Function]}
+          >
+            <label>
+              <input
+                aria-label="Select row 3"
+                checked={false}
+                disabled={true}
+                name="checkrow3"
+                onChange={[Function]}
+                type="checkbox"
+              />
+            </label>
+          </td>
+          <td
+            className=""
+            data-key={1}
+            data-label="Name"
+            onMouseEnter={[Function]}
+          >
+            Microsoft Azure DevOps
+            <span
+              data-testid="azure-devops-unsupported-automatic-revoke-tooltip"
+              data-tip="Provided API does not support the automatic token revoke. You can revoke it manually."
```

Comment:
Can we add URL in the tooltip that would be Azure DevOps specific?

Reply:
I have fixed it.